WalletWasabi.Fluent.Generators/Generators/AutoInterfaceGenerator.cs

```diff
@@ -102,20 +91,26 @@ property.SetMethod switch
 					let defaultValueString = defaultValue != null ? " = " + defaultValue : null
 					select $"{attributeList?.ToFullString()}{refKind}{type} {name}{defaultValueString}";
 
-				signature += string.Join(", ", parameters);
+				string methodSignature;
 
-				signature += ");";
+				if (!IsInterfaceMethodImplementation(method))
+				{
+					methodSignature = $"\t {returnType} {method.Name}{genericTypeOrEmpty}({string.Join(", ", parameters)});";
+				}
+				else
+				{
+					methodSignature = $"\t /* SKIPPED: Implements an interface */ /* {returnType} {method.Name}{genericTypeOrEmpty}({string.Join(", ", parameters)}); */";
```

Comment:
I would prefer not to generated commented code.

Instead, simply avoid calling `methods.Add()` in this case, IMO.